integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.html

```diff
@@ -14,10 +14,13 @@
 <ng-template #paymentData let-paymentLink="paymentLink">
   <div>
     <div
-      *ngIf="paymentLink.renderType === RENDER_PATTERN.FULL_PAGE"
+      *ngIf="
+        paymentLink.renderType === RENDER_PATTERN.FULL_PAGE &&
+        paymentLink?.destination?.url
+      "
       class="cx-payment-link"
     >
-      <a class="btn btn-primary" [href]="paymentLink.data">{{
+      <a class="btn btn-primary" [href]="paymentLink?.destination?.url">{{
```

Comment:
i am wondering if we should bypassSanitizer with this url? afaik we never got problem without it.

Reply:
note that when
paymentLink?.destination?.url exists, paymentLink.data has the same value as paymentLink.destination.url